docs/non-normative/db-migration.md

```diff
@@ -43,7 +43,7 @@ This section summarizes the changes made to the HTTP semantic conventions
 from
 [v1.24.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/database/README.md).
 to
-[v1.31.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.31.0/docs/database/README.md).
+[v1.33.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.33.0/docs/database/README.md).
```

Comment:
Adding a comment so we don't merge it accidentally.

the tag does not exist yet, so the link check fails. The plan is to merge it right before release and avoid breaking other PRs